library/network_connections.py

```diff
@@ -1630,7 +1646,12 @@ def _cmp(a, b):
             if t_b <= 0:
                 return -1
             return cmp(t_a, t_b)
-        cons.sort(cmp = _cmp)
+
+        if sys.version_info[0] == 3:
```

Comment:
There is also `Util.PY3` above. Either we make use of this `PY3` helper, or we drop it and use `sys.version_info[0]`.